stan/math/rev/fun/pow.hpp

```diff
@@ -117,12 +117,18 @@ inline var pow(const var& base, const var& exponent) {
  *
  * \f$\frac{d}{dx} \mbox{pow}(x, c) = c x^{c-1}\f$.
  *
+ * The template parameters are coded as they are so that arithmetic
+ * types will not be promoted into the `var` slots.
+ *
+ * @tparam Var var type
+ * @tparam Arith arithmetic type
  * @param base Base variable.
  * @param exponent Exponent scalar.
  * @return Base raised to the exponent.
  */
-template <typename Arith, require_arithmetic_t<Arith>...>
-inline var pow(const var& base, Arith exponent) {
+template <typename Var, typename Arith, require_var_t<Var>...,
+          require_arithmetic_t<Arith>...>
+inline var pow(const Var& base, Arith exponent) {
```

Comment:
I'm fine with merging as is. Should we make a separate issue to have all these be passed by value?

Reply:
Yes, that should definitely go  in a separate issue, along with performance evaluations.